build-logic/build.gradle.kts

```diff
@@ -21,8 +21,6 @@ repositories {
         url = URI("https://artifactory.terasology.io/artifactory/virtual-repo-live")
     }
 
-    // TODO MYSTERY: As of November 7th 2021 virtual-repo-live could no longer be relied on for latest snapshots - Pro feature?
-    // We've been using it that way for *years* and nothing likewise changed in the area for years as well. This seems to work ....
```

Comment:
@Cervator - Do you know if this comment is still relevant with the new artifactory?

Reply:
I haven't tested but we should try doing so. With a little luck we can undo the hack. It was a _huge_ upgrade over multiple major versions so it could well have changed the quirky bits a lot.

Reply:
if virtual repos do not work it is an artifactory bug. see feature matrix:
https://jfrog.com/help/r/jfrog-hosting-models-documentation/artifactory-comparison-matrix

Reply:
took out the TODO removal, as controversal at the moment.

Reply:
So the artifacts specific to this issue actually is snapshot-level builds of our project artifacts - game modules, libraries, etc. They wouldn't be available on the regular internet repos :-)

I think Maven Central is in there somewhere to resolve 3rd party libs. Artifactory may also _cache_ those - cannot quite recall the details.

Reply:
you are of course right, the standard repos are before. what distracted me was that the url you list:
https://artifactory.terasology.io/ui/native/virtual-repo-live

takes a while to load and lists internet as well. while i d have expected an url with terasoloty only.